src/hooks/useNpgSdk.ts

```diff
@@ -0,0 +1,60 @@
+import { useEffect, useState } from "react";
+import type {
+  FieldId,
+  FieldStatus,
+} from "../features/payment/components/IframeCardForm/types";
+import { getConfigOrThrow } from "../utils/config/config";
+import createBuildConfig from "../utils/buildConfig";
+
+export type SdkBuild = {
+  onChange?: (field: FieldId, fieldStatus: FieldStatus) => void;
+  onReadyForPayment?: () => void;
+  onPaymentComplete?: () => void;
+  onPaymentRedirect?: (urlRedirect: string) => void;
+  onBuildError: () => void;
+};
+
+const noop = () => {
+  // noop
+};
+
+export const useNpgSdk = ({
+  onChange = () => null,
+  onReadyForPayment = () => null,
+  onPaymentComplete = () => null,
+  onPaymentRedirect = () => null,
+  onBuildError,
+}: SdkBuild) => {
+  const [sdkReady, setSdkReady] = useState(false);
+
+  const createBuild = () => {
+    try {
+      // THIS is mandatory cause the Build class is defined in the external library called NPG SDK
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
```

Comment:
I think it's time to get rid of this @ts-ignore globally. To do that try to rename the file image.d.ts in modules.d.ts (or maybe globals.d.ts) ad append these lines.
`/** Build class defined in the external NPG SDK library */`
`declare const Build: any;`

Reply:
I've followed your suggestion and was able to drop the rule-disable comments